#! /usr/bin/env python

PACKAGE = "starrobot_calib"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()
gen.add("servo1",bool_t,20,"Enable servo1 test", False)
gen.add("servo2",bool_t,20,"Enable servo2 test", False)
gen.add("servo3",bool_t,20,"Enable servo3 test", False)
gen.add("servo4",bool_t,20,"Enable servo4 test", False)
gen.add("angle", int_t, 0, "Servo rotation angle value", 0,-180, 180)
exit(gen.generate(PACKAGE, "calibrate_servo", "CalibrateServo"))

